### PR TITLE
Initialize documentation for RobotFramework-TestsuiteManagement repo

### PR DESCRIPTION
**Pull request by namsonx**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -1,260 +1,186 @@
...
+`Holger Queckenstedt <Holger.Queckenstedt@de.bosch.com>`_
 
-License 
-~~~~~~~
+License
+-------
 
-robotframework-testsuitesmanagement is open source software provided under the `Apache License
-2.0`__. 
+`License: BIOSL v4 <http://bios.intranet.bosch.com/bioslv4-badge.svg>`_
```
Comment (test-fullautomation) [inline README.rst]: Hi Son,
the license information need to be corrected.
Please refer to the header of all files.
Thank you,
 Thomas

**Review by test-fullautomation — COMMENTED** [review]
Hi Son,
the docstrings look very nice now!
Thank you,
 Thomas

File: README.rst (hunk 1 above)
Comment (namsonx) [reply README.rst]: Hello Thomas,

    I updated the license in readme.rst as you mentioned.

Thank you,
Son

File: RobotFramework_Testsuites/Config/robot_schema.json [hunk 2]
```diff
@@ -1,16 +1,3 @@
-//  Copyright 2020-2022 Robert Bosch Car Multimedia GmbH
```
Comment (test-fullautomation) [inline RobotFramework_Testsuites/Config/robot_schema.json:1]: Hi Son,
very good. I overlooked this file. 
Here comments are not allowed of course.
Thank you,
 Thomas

**Review by test-fullautomation — APPROVED** [review]
Hi Son,
thank you for the pull request.
I will merge it now.
Thank you,
 Thomas

**Pull request merged**